### PR TITLE
Implement e2e tests with Cypress

### LINKED ISSUE #10: Implement e2e tests with Cypress
We need to install and configure [Cypress](https://www.cypress.io/) to create e2e tests.

- [ ] Install Cypress
- [ ] Configure Cypress
- [ ] Create at least one sample test

### PR DESCRIPTION
**Pull request by arianerocha**
PR to add: 

- Cypress as a dev dependency;
- Cypress configurations;
- Login and Register pages as sample tests.

Login spec ts

It closes #10

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "baseUrl":"http://localhost:3000",
```
Comment (rluders) [inline cypress.json:2]: missing space

File: cypress/integration/Auth/Register.spec.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { last } from "cypress/types/lodash";
```
Comment (rluders) [inline cypress/integration/Auth/Register.spec.ts:1]: If single quote, single quote for all

File: cypress/integration/index.spec.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+context('Sample test', () => {
```
Comment (rluders) [inline cypress/integration/index.spec.ts:1]: Do we need a sample test?

File: cypress/support/index.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import "./commands";
```
Comment (rluders) [inline cypress/support/index.ts:17]: Single quotes?

File: cypress/integration/Auth/Register.spec.ts (hunk 2 above)
Comment (arianerocha) [reply cypress/integration/Auth/Register.spec.ts:1]: **Prettier** should have done this.
I will look at how to apply the react npm script to do this.

File: cypress/integration/index.spec.ts (hunk 3 above)
Comment (arianerocha) [reply cypress/integration/index.spec.ts:1]: Ooops! I forgot to delete this, Tks

File: cypress/integration/Auth/Register.spec.ts (hunk 2 above)
Comment (rluders) [reply cypress/integration/Auth/Register.spec.ts:1]: https://prettier.io/docs/en/precommit.html

**Pull request merged**